R/teal_data.R

```diff
@@ -3,8 +3,12 @@
 #' @description `r lifecycle::badge("stable")`
 #' Universal function to pass data to teal application
 #'
-#' @param ... (`TealDataConnector`, `TealDataset`, `TealDatasetConnector`)\cr
-#'   objects
+#' @param ... (`TealDataConnector`, `TealDataset`, `TealDatasetConnector`, `any`)\cr
+#'  - When one of the `Teal*` objects are provided, then function returns `TealData` object.
+#'    This way of specifying data is deprecated and will be removed in the next release.
+#'  - From version 0.4.0, one can provide any object as a named argument and function will
+#'    return `teal_data` object. Objects provided in `...` will be stored in `teal_data` environment
+#'    under the same name as the argument name.
```

Comment:
This should only contain accepted argument types.

Reply:
There are no restrictions on accepted argument types. Technically it is limited by teal.slice only, but it shouldn't be a concern of this package

Reply:
I mean this is not the place to explain different return values. Sorry or not being clear.

Reply:
Please propose something which makes sense at this moment. Feel free to make a commit 👍